examples/run_examples.jl

```diff
@@ -22,8 +22,8 @@ scip_solver = JuMP.optimizer_with_attributes(SCIP.Optimizer)
 ecos_solver = JuMP.optimizer_with_attributes(ECOS.Optimizer, "verbose" => false, "maxit" => 10000)
 scs_solver = JuMP.optimizer_with_attributes(SCS.Optimizer)
 cplex_solver = JuMP.optimizer_with_attributes(CPLEX.Optimizer, "CPX_PARAM_SCRIND" => 0)
-couenne_solver = JuMP.with_optimizer(AmplNLWriter.Optimizer, "couenne.exe")
-bonmin_solver = JuMP.with_optimizer(AmplNLWriter.Optimizer, "bonmin.exe")
+couenne_solver = JuMP.with_optimizer(AmplNLWriter.Optimizer("couenne.exe"))
+bonmin_solver = JuMP.with_optimizer(AmplNLWriter.Optimizer("bonmin.exe"))

```

Comment:
Should avoid the use of `with_optimizer`, I think new syntax in this case is `couenne_solver = () -> AmplNLWriter.Optimizer("couenne.exe")`

Reply:
Noted. It looks like these solvers weren't being used, so I also commented them out.